core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20210615_sharded_entity_tables.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="davidan" id="create_sharded_entity_tables">
+        <sql stripComments="true">
+            -- TODO: davidan turn the create-table code into a stored proc; call the stored proc repeatedly
```

Comment:
What happens if we tried 16 shards and then went "actually, we need more!" - would it be easy to increase the number of shards? Would it be worth it to make that process easy?

Reply:
creating the additional tables is pretty trivial, but re-allocating the existing rows into their appropriate shards is a big pain - it needs downtime and is a tricky SQL operation. I'd much rather overestimate the number of shards and avoid a second data migration.